### DISCUSSION TITLE
[Esercitazione 6 Dicembre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#339] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 1**. Si consideri il sistema omogeneo di 4 equazioni in 5 incognite

$$\begin{cases}x_1+x_2+2x_3+x_5=0\\
x_1-x_2+x_4+x_5=0\\
x_1-x_3-x_5=0\\
x_1+2x_2+5x_3+3x_5=0\end{cases}$$

Sia $\Sigma_0$ l'insieme delle soluzioni. Determinare una matrice $A$ tale che $\Sigma_0=Ker\ A.$
Applicare il metodo di Gauss e determinare un sistema omogeneo a scala $S\underline x=\underline 0$, equivalenete al sistema dato.
Determinare una base per il sottospazio $W$ uguale allo Span delle colonne di $S$ .
Determinare una base per $Im\ L_S$ .
Determinare una base per $\Sigma_0$ : più precisamente determinare $l\in \mathbb{N}$ e vettori $\\{w_1,...,w_l\\}$ in $\mathbb{R}^5$ linearmente indipendenti tali che $\Sigma_0=Span(w_1,...,w_l)$.
Determinare una base per il sottospazio uguale allo Span delle colonne di $A$.

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/438fbc7a-bf0b-46ce-a5a4-66757e856c13)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/8d39407e-6994-447f-bab2-77c12666093f)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/cab67c75-5030-44d8-974f-b1105a33d5a8)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/32b569a6-2ca3-4c13-b5b9-0d24d986fb43)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b0122dd5-70c3-4c0a-a59e-d68ef8876109)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/14f70681-ab99-4d5c-a3df-41ed219acc5a)

**Comment by daromacarlo** [comment]
Ho un dubbio riguardo il punto 3 e tutte le richieste analoghe che ci sono nei vari fogli di esercizi, posso determinare "a occhio" una base senza sapere la Dim del sottospazio?
![Immagine 2023-12-07 175614](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/d2b6ff29-4068-4ea3-a4e4-aad3a1574955)
![Immagine 2023-12-07 175630](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4399a315-317b-4831-8dac-d9b684b0b705)
![Immagine 2023-12-07 175652](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/155868a3-2bce-4b83-b427-32488b634e24)
![Immagine 2023-12-07 175712](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/ee8bfed5-bab7-4ec1-8afc-c65fa46ec135)
![Immagine 2023-12-07 175728](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/989d3a69-4679-4435-a86c-224f6a68e97a)
![Immagine 2023-12-07 175751](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/4f98634e-d2ec-45d3-a790-6f7f52a9a78a)
![Immagine 2023-12-07 175812](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/a9b66574-1dd7-401f-a3a6-b213347ac96e)
![Immagine 2023-12-07 175822](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/f5d814af-2843-4047-b524-0bd3535c6679)

**Comment by CiottoloMaggico** [comment]
**Soluzione di esercizio dato in classe**
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/3577fd77-a1ba-4c82-b37d-83b339d87e35)
**Soluzione esercizio dato online**
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/52caaaa5-7497-444e-8fbc-63b27007c8a3)

**Comment by daromacarlo** [comment]
@CiottoloMaggico mi riferivo al punto che chiede di trovare la base del sottospazio W

**Comment by CiottoloMaggico** [comment]
@CarloDaRomadev Il sottospazio W è Im(S) o Im(Ls).
Ls : R^n -> R^m è per definizione lo span delle colonne di S.

**Comment by daromacarlo** [comment]
@CiottoloMaggico ho capito, quando mi chiede di trovare una base di W, prendo i vettori dove compaiono i pivot perché sono tra loro L.ind.

**Comment by daromacarlo** [comment]
@CiottoloMaggico effettivamente così non li trovo a "occhio"

**Comment by CiottoloMaggico** [comment]
@CarloDaRomadev in queste cose niente è ad "occhio" ahahahah

**Comment by daromacarlo** [comment]
@CiottoloMaggico esatto, per questo chiedevo, grazie Federì
